packages/calcite-components/src/components/list-item/list-item.e2e.ts

```diff
@@ -396,6 +396,28 @@ describe("calcite-list-item", () => {
     expect(calciteListItemToggle).toHaveReceivedEventTimes(0);
   });
 
+  it("should set displayMode on slotted list", async () => {
+    const page = await newE2EPage({
+      html: html`<calcite-list-item><calcite-list></calcite-list></calcite-list-item>`,
+    });
+
+    const listItem = await page.find("calcite-list-item");
+    const list = await page.find("calcite-list");
+
+    expect(await listItem.getProperty("displayMode")).toBe("flat");
+    expect(await list.getProperty("displayMode")).toBe("flat");
+
+    listItem.setProperty("displayMode", "nested");
+    await page.waitForChanges();
+
+    expect(await list.getProperty("displayMode")).toBe("nested");
+
+    listItem.setProperty("displayMode", "flat");
+    await page.waitForChanges();
+
+    expect(await list.getProperty("displayMode")).toBe("flat");
+  });
+
```

Comment:
Should we add a 3rd nesting level to the test?  I assume since the `getListItemChildren` util is flattening everything it should work just fine, but might be cool to have a 3rd layer in here to assert on.

Reply:
Since the list-item is setting it, we wouldn't need to test another level.